generate-stackbrew-library.sh

```diff
@@ -4,6 +4,7 @@ set -eu
 declare -A aliases=(
 	[3.11]='3'
 	[4.0]='4 latest'
+	[4.1]=`4 beta`
```

Comment:
https://github.com/docker-library/cassandra/actions/runs/3206451808/jobs/5240216296#step:3:45 :joy: :sweat_smile:

However, the intent here isn't right either because we don't want the `4` alias to point to the new beta release. :see_no_evil:

```suggestion
```

(Otherwise, this PR looks fine! :+1:)

Reply:
I wasn't sure what should happen here, so thank you very much for the review!